icecandidate.go

```diff
@@ -70,6 +72,34 @@ func newICECandidateFromICE(i ice.Candidate) (ICECandidate, error) {
 	return c, nil
 }
 
+// newIdentifiedICECandidateFromICE creates a new ICECandidate with the specified media stream identification tag and media description index.
+func newIdentifiedICECandidateFromICE(candidate ice.Candidate, sdpMid string, sdpMLineIndex uint16) (ICECandidate, error) {
+	c, err := newICECandidateFromICE(candidate)
+	if err != nil {
+		return ICECandidate{}, err
+	}
+
+	c.SDPMid = sdpMid
+	c.SDPMLineIndex = sdpMLineIndex
+
+	return c, nil
+}
+
+// newICECandidatesFromICE creates ICECandidates from ice.Candidates with the specified media stream identification tag and media description index.
+func newIdentifiedICECandidatesFromICE(iceCandidates []ice.Candidate, sdpMid string, sdpMLineIndex uint16) ([]ICECandidate, error) {
```

Comment:
Again, why a new function?  BTW, the comment is incorrect, you didn't update the function name.